sklearn/linear_model/tests/test_coordinate_descent.py

```diff
@@ -712,3 +712,15 @@ def test_enet_float_precision():
             assert_array_almost_equal(intercept[np.float32],
                                       intercept[np.float64],
                                       decimal=4)
+
+    def test_enet_l1_ratio():
+        # Test that we do not allow l1_ratio = 0 in
+        # ElasticNetCV and MultiTaskElasticNetCV
+        X = np.array([[1, 2, 4, 5, 8], [3, 5, 7, 7, 8]]).T
+        y = np.array([12, 10, 11, 21, 5])
+        clf = ElasticNetCV(l1_ratio=0)
+        clfl = ElasticNetCV(l1_ratio=[0, 0.5])
+        clfm = MultiTaskElasticNetCV(l1_ratio=0)
+        assert_raises(ValueError, clf.fit, X, y)
```

Comment:
Can you use ``assert_raise_message`` to check the error message? Apart from that LGTM.